examples/react-webpack/src/index.js

```diff
@@ -27,25 +27,7 @@ const ProfileBox = () => {
   return <h3>{name}</h3>
 }
 
-const AccreditationBox = () => {
-  const { loginStatus, getAccreditations } = useParallel()
-  const [response, setResponse] = useState(null)
-
-  useEffect(() => {
-    // if the user is now connected, then fetch accreditation info
-    if (loginStatus?.status === 'connected') getAccreditations?.().then(setResponse)
-  }, [loginStatus])
-
-  if (!response) return null
-
-  if (response.accreditations.some(({ status }) => status === 'current')) {
-    return <div>You are accredited!</div>
-  } else {
-    return <div>You are not accredited :(</div>
-  }
-}
```

Comment:
Should we include a call to `getProfile()` to then show the Parallel ID here?  Could be a good place to demonstrate fetching it.

Reply:
@bmuller This code already included a `ProfileBox` component which demonstrates the `getProfile` API. This led me to clean it up just a tad though: ffd46987ef09223b0be17318ddddeebf4afc611b